compiled_starters/cpp/spawn_redis_server.sh

```diff
@@ -6,6 +6,9 @@
 #
 # DON'T EDIT THIS!
 set -e
-cmake . >/dev/null
-make >/dev/null
-exec ./server "$@"
+export PATH=/cmake-portable/bin:$PATH
```

Comment:
Is this something that'd trigger on a user's machine? If not, can we just update PATH in the dockerfile?